src/main/features/response/views/confirmation/full-defence.njk

```diff
@@ -1 +1,6 @@
-<p>{{ t('We’ve emailed {{ claimant }} your response, explaining why you reject the claim.', { claimant: claim.claimData.claimant.name }) }}</p>
+{% if claim.response.defenceType === DefenceType.ALREADY_PAID %}
+  <p>{{ t('You told us you’ve paid the full claim amount of {{ amount }}. We’ve sent {{ claimant }} this response.',
```

Comment:
This could be greater than the full claim amount. Check with PO?

Reply:
spoke with content and PO. have updated the content.